src/pages/academy/blog/posts/preview/index.tsx

```diff
@@ -128,6 +129,8 @@ const BlogPreview = () => {
         imgSrcMobile: footer_banner_data?.mobile_banner_image?.id,
     }
 
+    const written_by_text: TString = '_t_Written by_t_'
+
```

Comment:
Please move this out of the compoents, no need for it to be here inside of the component.